microsoft-edge/webview2/releasenotes.md

```diff
@@ -20,6 +20,50 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
 > [!IMPORTANT]
 > While WebView2 is a preview, the .NET APIs are in the `prerelease package`.  
 
+## 1.0.627-prerelease  
+
+Release Date: Oct 19, 2020  
+
+[NuGet package][NuGetGallery1.0.672-prerelease] \| minimum Microsoft Edge version 88.0.672.0.
+
+#### General
+
+*    Added [WebView2 Group Policies][DeployedgeMicrosoftEdgeWebviewPolicies]. Navigate to [group policies for WebView2][Webview2ConceptsEnterpriseGroupPoliciesForWebview2] for best practice.
+*    Added [NavigateWithWebResourceRequest][ReferenceWin3210672Icorewebview2experimentalNavigateWithWebResourceRequest] method that lets you provide post data or additional request headers during navigation.
+*    Added [DOMContentLoaded][ReferenceWin3210672Icorewebview2experimentalAddDomcontentloaded] event that fires when the initial html document has been parsed.
+*    Added [Environment][ReferenceWin3210672Icorewebview2experimentalGetEnvironment] property on WebView2 to expose the WebView2 environment used to create such WebView2.
+*    Added [cookie management][ReferenceWin3210672Icorewebview2experimentalGetCookiemanager] APIs that allows end developers to authenticating the WebView session, or retrieving cookies from WebView to authenticate other tools. We are planning to make language/framework specific improvements; please navigate to [API Review: Cookie Management][GithubMicrosoftedgeWebview2AnnouncementIssue2] for more information.
+*    Updated [WebResourceResponseReceived][ReferenceWin3210672Icorewebview2experimentalAddWebresourceresponsereceived] event including adding immutable [WebResourceResponseView][ReferenceWin3210672Icorewebview2experimentalwebresourceresponseview] and [WebResourceResponseReceivedEventArgs::PopulateResponseContent][ReferenceWin3209628Icorewebview2experimentalwebresourceresponsereceivedeventargsPopulateresponsecontent] becoming [WebResourceResponseView::GetContent][ReferenceWin3210672Icorewebview2experimentalwebresourceresponseviewGetcontent].
+*    Disabled [Microsoft Defender Application Guard (WDAG)][WindowsSecurityThreatProtectionMicrosoftDefenderApplicationGuardWindows10] in WebView2.
+* Added [SystemCursorId](link to ICoreWebView2Controller.SystemCursorId) for Visual Hosting.
+* Bug fixes for IME issues in Visual Hosting.
```

Comment:
Can you elaborate what issues are fixed? Maybe say IME is now supported in visual hosting?

Reply:
@liminzhu - would "Input Method Header is now supported in visual hosting" work?

Reply:
A couple odd ones that are probably not worth elaborating here.